src/appleseed-max-impl/appleseedrenderer/projectbuilder.cpp

```diff
@@ -1725,15 +1725,15 @@ namespace
     {
         switch (filter_type)
         {
-          case 0:  return "blackman-harris";
-          case 1:  return "box";
-          case 2:  return "catmull";
-          case 3:  return "bspline";
-          case 4:  return "gaussian";
-          case 5:  return "lanczos";
-          case 6:  return "mitchell";
-          case 7:  return "triangle";
-          default: return "box";
+        case 0:  return "blackman-harris";
```

Comment:
I think two space indentation is missing here.

Reply:
True, the auto indent struck again.